### PR TITLE
Feature/catkin package

### PR DESCRIPTION
**Pull request by yvaind**
Minimal modifications to make osqp a catkin package. Since qdldl is included as submodule, there is no need to clone it when using osqp. Aside: I actually did not modify docs/make.bat - no idea why git is seeing changes.

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<package format="2">
+  <name>osqp</name>
+  <version>0.0.0</version>
+  <description>osqp</description>
+  <maintainer email="yvaind@ethz.ch">Yvain de Viragh</maintainer>
+  <license>TODO</license>
```
Comment (yvaind) [inline package.xml]: @mbjelonic What should the license be?

**Comment by remod** [comment]
Did you make it executable? Could that be the change to .bat?

File: package.xml (hunk 1 above)
Comment (mbjelonic) [reply package.xml]: The license is "Apache 2.0".

**Comment by yvaind** [comment]
@remod If at all, I did not make it executable intentionally. But it's an interesting hint, thanks! In between I have found what the changes actually are: A conversion between CRLF and LF line endings. I have spent quite some time trying to understand and fix the underlying issues (which are related to git itself), but to be honest I can still not guarantee that something similar might not again happen (at least when someone pulls form oxfordcontrol/osqp). Probably we would want to change "* text=auto" to "* -text" in .gitattributes, configure core.autocrlf to "input" instead of "false", and (even more questionable) configure core.eof to "LF". Not helpful at all in this story is that git is seemingly not able to display line endings online.

**Comment by mbjelonic** [comment]
I fixed the issue with the docs/make.bat file.

**Review by mbjelonic — APPROVED** [review]

**Pull request merged**